tmexp/io_constants.py

```diff
@@ -1,5 +1,5 @@
 from collections import Counter
```

Comment:
We should probably split this file in 2: IO constants could stay there, and data classes could go to `data.py`

Reply:
yeah I was thinking of that as well, it's getting a bit ugly :p